### PR TITLE
Add int8 as known go type

### PR DESCRIPTION
**Pull request by FoGhost**
- Add int8 to KnownTypeMap (the collection of known Go types.)
- Add Make help command

### CODE REVIEW

File: templates/yo_package.go.tpl [hunk 1]
```diff
@@ -5,7 +5,9 @@ package {{ .Package }}
 import (
 	"context"
 	"fmt"
+	"time"
```
Comment (kazegusuri) [inline templates/yo_package.go.tpl]: why are these packages required?

Comment (FoGhost) [reply templates/yo_package.go.tpl]: Oh.. no need this package now. I will delete it

Comment (FoGhost) [reply templates/yo_package.go.tpl]: @kazegusuri Fixed it, please check again later

File: templates/yo_package.go.tpl [hunk 2]
```diff
@@ -6,6 +6,7 @@ import (
 	"context"
 	"fmt"
 
+	"google.golang.org/grpc/codes"
```
Comment (kazegusuri) [inline templates/yo_package.go.tpl]: what about this

Comment (FoGhost) [reply templates/yo_package.go.tpl]: https://github.com/mercari/yo/blob/master/templates/type.go.tpl#L144

We use grpc `codes.xx` in yo template file. 
goimports somthing import a wrong codes package `grpc.go4.org/codes`.
So I import the dependent package strictly here.

Comment (kazegusuri) [reply templates/yo_package.go.tpl]: oh I see. nice

**Pull request merged**